Performance.md

```diff
@@ -481,6 +481,31 @@ Sometimes TypeScript's type-checking can be impacted by computationally intensiv
 This is rare, but can happen.
 Upgrading to a newer version of TypeScript (which can be more efficient) or to a newer version of an `@types` package (which may have reverted a regression) can often solve the issue.
 
+## Performance Tracing
+
+You might find that these diagnostic methods are too opaque to get a sense of why TypeScript feels slow.
+TypeScript provides a `--generateTrace` option that can give you a sense of the work the compiler is spending time on.
+`--generateTrace` will create an output file that can be analyzed within Edge or Chrome.
+
+Make sure TypeScript can compile your project without errors.
```

Comment:
It actually doesn't matter if there are errors (though you'll be measuring something different).

Reply:
```suggestion
Ideally, TypeScript will be able to compile your project without any errors, though it's not a strict requirement for tracing.
```